example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -11,6 +11,8 @@ controllers:
     concurrentSyncs: 20
     syncPeriod: 1h
     retryDuration: 24h
+  plant:
+    syncPeriod: 10s
```

Comment:
You also have to add `concurrentSyncs`, otherwise this default example won't work.
As the default in the code is `30s`, please also change `syncPeriod: 30s`.